releases.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
Why is this an array? For each release, we can just update two attributes, `commit_sha` and `version`. Once it is updated, that PR can be merged. There is no harm in keeping an array, except the script has some negligible extra work of picking up the first element.